cmd/jaeger/internal/extension/jaegerstorage/converter/writer.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package converter
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	otlp2jaeger "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+
+	spanstore_v1 "github.com/jaegertracing/jaeger/storage/spanstore"
+	"github.com/jaegertracing/jaeger/storage_v2/spanstore"
+)
+
+type TraceWriter struct {
+	spanWriter spanstore_v1.Writer
+}
+
+func NewTraceWriter(spanWriter spanstore_v1.Writer) (spanstore.Writer, error) {
```

Comment:
```suggestion
func NewTraceWriter(spanWriter spanstore_v1.Writer) spanstore.Writer {
```
don't declare error if it's never returned, it just introduces unnecessary error handling statements and makes it harder to maintain code coverage.